.github/workflows/conda_and_docs_build.yml

```diff
@@ -103,5 +103,5 @@ jobs:
       with:
         message: Update documentation
         branch: gh-pages
-        dir: docs/build
+        dir: docs/build/nightly
```

Comment:
What does this do? Does it just push the nightly folder and its contents, or just the contents?

Reply:
contents